changelog-generator/app/config-git.sh

```diff
@@ -3,4 +3,4 @@
 echo "Configuring git..."
 git config --global user.email "kyma.bot@sap.com"
 git config --global user.name "Kyma Bot"
-git config --global core.sshCommand 'ssh -i '$SSH_FILE''
+git config --global core.sshCommand 'ssh -i '"$SSH_FILE"''
```

Comment:
These single quotes around $SSH_FILE are needed? For sure at the end is too many by one.